src/styles/toolbar.m.css

```diff
@@ -37,3 +38,13 @@
 .runIconDisabled {
 	background-image: url("data:image/svg+xml;charset=utf-8,%3Csvg%20fill%3D%22%233C424E%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20xmlns%3Axlink%3D%22http%3A%2F%2Fwww.w3.org%2F1999%2Fxlink%22%20version%3D%221.1%22%20x%3D%220px%22%20y%3D%220px%22%20viewBox%3D%220%200%20100%20100%22%20style%3D%22enable-background%3Anew%200%200%20100%20100%3B%22%20xml%3Aspace%3D%22preserve%22%3E%3Cg%3E%3Cpath%20d%3D%22M29.6%2C84.7l48.3-28.6c4.6-2.7%2C4.6-9.4%2C0-12.1L29.6%2C15.3C24.9%2C12.5%2C19%2C15.9%2C19%2C21.4v57.3C19%2C84.1%2C24.9%2C87.5%2C29.6%2C84.7z%22%3E%3C%2Fpath%3E%3C%2Fg%3E%3C%2Fsvg%3E");
 }
+
+.consoleIconEnabled {
+	background-image: url("data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiIHN0YW5kYWxvbmU9Im5vIj8+CjwhRE9DVFlQRSBzdmcgUFVCTElDICItLy9XM0MvL0RURCBTVkcgMS4xLy9FTiIgImh0dHA6Ly93d3cudzMub3JnL0dyYXBoaWNzL1NWRy8xLjEvRFREL3N2ZzExLmR0ZCI+Cjxzdmcgd2lkdGg9IjEwMCUiIGhlaWdodD0iMTAwJSIgdmlld0JveD0iMCAwIDY0IDY0IiB2ZXJzaW9uPSIxLjEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiIHhtbDpzcGFjZT0icHJlc2VydmUiIHhtbG5zOnNlcmlmPSJodHRwOi8vd3d3LnNlcmlmLmNvbS8iIHN0eWxlPSJmaWxsLXJ1bGU6ZXZlbm9kZDtjbGlwLXJ1bGU6ZXZlbm9kZDtzdHJva2UtbGluZWpvaW46cm91bmQ7c3Ryb2tlLW1pdGVybGltaXQ6MS40MTQyMTsiPgogICAgPGcgdHJhbnNmb3JtPSJtYXRyaXgoMC45ODUzNDIsLTAuOTg2MTUxLDAuOTg2MTUxLDAuOTg1MzQyLC0yMy40OTc2LDE3LjQ0NjEpIj4KICAgICAgICA8cGF0aCBkPSJNMTkuNTcxLDExLjU0OUMxOS41NzEsMTAuNzg0IDE4Ljk1MSwxMC4xNjMgMTguMTg2LDEwLjE2M0wxNS40MTQsMTAuMTYzQzE0LjY1LDEwLjE2MyAxNC4wMjksMTAuNzg0IDE0LjAyOSwxMS41NDlMMTQuMDI5LDMyLjY4QzE0LjAyOSwzMy40NDUgMTQuNjUsMzQuMDY2IDE1LjQxNCwzNC4wNjZMMTguMTg2LDM0LjA2NkMxOC45NTEsMzQuMDY2IDE5LjU3MSwzMy40NDUgMTkuNTcxLDMyLjY4TDE5LjU3MSwxMS41NDlaIiBzdHlsZT0iZmlsbDpyZ2IoNTUsMTgyLDI0Nyk7Ii8+CiAgICA8L2c+CiAgICA8ZyB0cmFuc2Zvcm09Im1hdHJpeCgtMC4wMDAzMDM2MDUsLTEuMzk0MDYsMS4zOTQwNiwtMC4wMDAzMDM2MDUsMTQuNDE4MSw3NC4wOTA1KSI+CiAgICAgICAgPHBhdGggZD0iTTE5LjU3MSwxMS41NDlDMTkuNTcxLDEwLjc4NCAxOC45NTEsMTAuMTYzIDE4LjE4NiwxMC4xNjNMMTUuNDE0LDEwLjE2M0MxNC42NSwxMC4xNjMgMTQuMDI5LDEwLjc4NCAxNC4wMjksMTEuNTQ5TDE0LjAyOSwzMi42OEMxNC4wMjksMzMuNDQ1IDE0LjY1LDM0LjA2NiAxNS40MTQsMzQuMDY2TDE4LjE4NiwzNC4wNjZDMTguOTUxLDM0LjA2NiAxOS41NzEsMzMuNDQ1IDE5LjU3MSwzMi42OEwxOS41NzEsMTEuNTQ5WiIgc3R5bGU9ImZpbGw6cmdiKDU1LDE4MiwyNDcpOyIvPgogICAgPC9nPgogICAgPGcgdHJhbnNmb3JtPSJtYXRyaXgoMC45ODQ5ODIsMC45ODY1MDksLTAuOTg2NTA5LDAuOTg0OTgyLDIwLjEzNjEsMi40NjkxOSkiPgogICAgICAgIDxwYXRoIGQ9Ik0xOS41NzEsMTEuNTQ5QzE5LjU3MSwxMC43ODQgMTguOTUxLDEwLjE2MyAxOC4xODYsMTAuMTYzTDE1LjQxNCwxMC4xNjNDMTQuNjUsMTAuMTYzIDE0LjAyOSwxMC43ODQgMTQuMDI5LDExLjU0OUwxNC4wMjksMzIuNjhDMTQuMDI5LDMzLjQ0NSAxNC42NSwzNC4wNjYgMTUuNDE0LDM0LjA2NkwxOC4xODYsMzQuMDY2QzE4Ljk1MSwzNC4wNjYgMTkuNTcxLDMzLjQ0NSAxOS41NzEsMzIuNjhMMTkuNTcxLDExLjU0OVoiIHN0eWxlPSJmaWxsOnJnYig1NSwxODIsMjQ3KTsiLz4KICAgIDwvZz4KPC9zdmc+Cg==");
```

Comment:
I have been using https://jakearchibald.github.io/svgomg/ to ensure that my SVG is as optimized as possible... also, base64 is a lot heavier than it needs to be from a bandwidth perspective.  I would recommend just URL encoding it (https://www.urlencoder.org/).